src/cli.ts

```diff
@@ -38,15 +39,19 @@ if (args['--help']) {
     Options
       --version, -v       Version number
       --help, -h          Displays this message
+
+      --gitignore         Respect ignore patterns in '.gitignore' files
 `);
 
   process.exit(0);
 }
 
 const [pattern] = args._.slice(0, 1);
+const gitignore = args['--gitignore'];
```

Comment:
I would try to make sure only the value `"true"` works, othewise you would pass `"false"` which will be evaluated to `true` (because it's a string).

This is an example to something that should work as I mentioned
```ts
const gitignore = args['--no-gitignore'] !== "true";
```

Reply:
This is where CLI parsers like `yargs` do a pretty good job, as boolish parsing CLI arguments is not that simple. There are many variations possible:

--(no-)gitignore
--(no-)gitignore=true/false
--(no-)gitignore true/false

Negations don't make it easier, to say it the least... Especially as you have to understand how globby deals with the `gitignore` option in the first place.

The current way this PR treats it, was the most logical to me. If at all I would recommend naming the CLI option:

`--gitignore`

And have it's default set to `true` so you can use:

`--gitignore=false` or `--gitignore false`

And then introduce a shortcut, called:

`--no-gitignore`

Ar least thus is how `yargs` treats it and for me it would make things more clear.

Reply:
Yeah, the problem with yargs that [it's heavy](https://packagephobia.com/result?p=yargs) and an overkill for the use of this package. I actually prefer a single way to do this.

I think that having `--gitignore=true` accepting files from within the `.gitignore` file sounds reasonable. While no arguments or "false" (which doesn't really need a special treatment) means to not accept those files. I understand that the [globby default is different](https://github.com/jamiebuilds/globby-cli/blob/82ac1e620e53d91bfe1382142f31e6b2afeb1e28/cli.js#L16) but I'm not willing to change the default, I think that it works great for most usecase of this tool.

Reply:
Fully agreed, good discussion. Let me change it accordingly and amend my commit. I'll do it later this day.

Just a sidenote (without any further implication to our discussion): I agree to what you said about `yargs` beeing "to heavy", thus I respect your decision, not wanting it to "bloat up" your library with transitive dependencies of `yargs`. If everyone would think that way, the NPM package ecosystem would be way healthier. However for me it much depends on what use case you have for your library. As I see it, your library is going to be used by "end-users" mostly, as those who need globbing functionallity will use `globby` or any other glob library in the first place. Thus, seen as an application-type package, I don't think the amount of transitive dependencies really matters, if it is not for the speed of installation and any security considerations in future, if any of the dependencies gets outdated, orphphaned or voulnerable. But as said, I will go ahead and implement it as discussed, it doesn't make sense to use `yargs` or some oder CLI library like `meow` for some boolish logic... (however beeing close ;) ).

Let me amend the commit in this PR later! Thanks for your quick reply!

Reply:
It matters because I expect `npx globitor` to be really fast, and adding dependencies would make it slower.
![image](https://user-images.githubusercontent.com/11733036/130359251-0ff4c20b-5537-44da-bd13-41776adce895.png)